assets/css/_pages.scss

```diff
@@ -1,3 +1,33 @@
 span.highlight {
     background-color: yellow;
 }
+
+.historical-records {
+  table {
+    width: auto;
+    margin: $small-spacing 0;
+    table-layout: fixed;
+    border-collapse: collapse;
+  }
+
+  table:first-child {
+    width: 100%;
+  }
+
+  th {
+    padding: $small-spacing;
+    font-weight: 600;
+    color: transparentize(#ffffff, .3);
```

Comment:
Color `#ffffff` should be written as `#fff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexlength)